File: Data Structures & Algorithms/concatenation-of-array/submission-0.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> result; 
        int len = nums.size(); 
        for(int i = 0; i < len * 2; i++) {
            result.push_back(nums[i%len]); 
        }
        return result; 
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} k
     * @return {number[]}
     */
    topKFrequent(nums, k) {
        const hash = new Map(); 
        const result = []; 
        for(let num of nums) {
            let count = (hash.get(num) || 0 ) + 1; 
            hash.set(num, count); 
        }
        while(k--) {
            let maxCount = -1; 
            let ans; 
            for(let [key, value] of hash.entries()) {
                if (value > maxCount ){ 
                    maxCount = value; 
                    ans = key; 
                }
            }
            result.push(ans); 
            hash.set(ans, 0); 
        }
        return result; 
    }

}
